packages/jest-serializer/src/__tests__/index.test.ts

```diff
@@ -16,7 +16,7 @@ const objs = [
   null,
   [0, true, '2', [3.14, {}, null]],
   {key1: 'foo', key2: 'bar', key3: {array: [null, {}]}},
-  {minusInf: -Infinity, nan: NaN, plusInf: +Infinity},
+  {minusInf: Number(Infinity), nan: NaN, plusInf: +Infinity},
```

Comment:
negative infinity is a valid thing
you could put here `Number.NEGATIVE_INFINITY`

Reply:
sure